sub-packages/bionemo-esm2/tests/bionemo/esm2/model/finetune/test_finetune.py

```diff
@@ -54,6 +54,7 @@ def pretrain_data_module(dummy_protein_dataset, dummy_parquet_train_val_inputs):
         micro_batch_size=4,
         min_seq_length=None,
         max_seq_length=1024,
+        num_workers=1,
```

Comment:
Why not 0? If you get a pin error you could add a `pin_memory=num_workers>0` in your datamodule kind of thing.

Reply:
It interleaves data loading and computation, and might catch multiprocessing bugs? But I'm not picky either way

Reply:
the benefit to 0 in testing would be that you can run pdb and land in a data thread if that's where an error happens. I guess you're right that it has that drawback on not testing a standard case as well though.

Reply:
whatever you want to do is fine here.

Reply:
yeah let's keep it at 1 for now -- if we do get a reproducible error here, we can always set it to zero for debugging.